### DISCUSSION TITLE
Projects Insights: graph groups by iteration name, not iteration identity

### DISCUSSION THREAD

**Discussion by hngl** [community/community#46728] [Projects and Issues]
### Select Topic Area

Bug

### Body

Hi,

We are happily using the Issues almost a year now. We are planning issues for iteration "Sprint 13–14" (dates Mar 24 – Apr 6, 2023) now. However these issues show up in the graph under "Sprint 13–14" (dates Mar 25, 2022 – Apr 7, 2022).

Graph showing incomplete issue at past Sprint:
<img width="1031" alt="Graph showing issue at wrong iteration" src="https://user-images.githubusercontent.com/15790266/217500378-31169d30-2ede-40c9-baa3-1071654f0067.png">

Looking at the issue details, it is clearly assign to the future "Sprint 13–14".
<img width="204" alt="Issue detail showing assigned iteration" src="https://user-images.githubusercontent.com/15790266/217502757-2a65f3d1-e47b-4c2e-98bc-b3aa094f7f00.png">

We can rename past iterations or archive old items. But it would be nice if that would not be necessary.

Kind regards,
Jurriaan

**Comment by hngl** [comment]
Somewhat related to, but not similar to, https://github.com/community/community/discussions/43186

**Comment by chiedo** [comment]
@hngl, thanks for calling this out! I'll share it with the team! 🙇🏿
